less/mixins.less

```diff
@@ -0,0 +1,50 @@
+// Mixins below variables
+.width-100 {
+  width: 100%;
+}
+.center() {
```

Comment:
Mixins are not necessary for something this simple usually. You don't save much time or add much functionality using a mixin versus the normal line here.